bedrock/newsletter/templates/newsletter/family.html

```diff
@@ -0,0 +1,45 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends 'base-protocol.html' %}
+
+{% block page_title %}Firefox heart‘s all families (and teachers too){% endblock page_title %}
+
+{% block page_desc %}{% endblock %}
+
+{% block body_class %}{{ super() }} newsletter-family{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('protocol-newsletter') }}
+  {{ css_bundle('newsletter-family') }}
+{% endblock %}
+
+{% block content %}
+<main class="newsletter-main" role="main">
+  {% call split(
+    image_url='img/newsletter/family/illustration.svg',
+    media_class='mzp-l-split-v-start',
+    block_class='mzp-t-content-lg'
+  ) %}
+    {% set icon_heart = '<img src="' ~ static('img/newsletter/family/heart.svg') ~ '" width="36" height="36" alt="heart">' %}
+    {% set newsletter_title = 'Firefox ' ~ icon_heart ~'‘s all families (and teachers too)' %}
+    {{ email_newsletter_form(
+      title=newsletter_title|safe,
+      desc='Firefox is made in part by lots of real-life, tired yet strangely optimistic parents, and we’re backed by an awesome non-profit too.',
+      newsletters='mozilla-foundation, mozilla-and-you',
+      button_class='mzp-t-dark',
+      include_title=True
+    ) }}
```

Comment:
**to confirm**: this is meant for a specific en-US audience and US only location? if so, maybe we can use `include_country=False` and `include_language=False` to reduce form length? (or hide those fields instead of removing them altogether, as the default values seem to be what we want)

Reply:
I'm gonna hang fire on removing things until I hear back from stakeholders on what the requirements are. They may also want to remove the multi-newsletter option too. Good call out still!